app/src/main/kotlin/com/wire/android/ui/home/HomeDrawer.kt

```diff
@@ -100,7 +98,14 @@ fun HomeDrawer(
 
         Spacer(modifier = Modifier.weight(1f))
 
-        val bottomItems = listOf(Settings, Support, Debug)
+        val bottomItems = buildList {
+            if (LocalFeatureVisibilityFlags.current.Settings) {
+                add(Settings)
+            }
```

Comment:
As said in the other comment. It's not the whole Settings screen that is supposed to be hidden, right? It's just the `App Settings` section INSIDE the main `Settings` screen.